README.md

```diff
@@ -24,14 +24,15 @@ Use `riviere` if you want an easy way to log all the HTTP traffic for your serve
     3. [inbound.request.enabled](#options_inbound_request_enabled)
     4. [bodyKeys](#options_body_keys)
     5. [bodyKeysRegex](#options_body_keys_regex)
-    6. [color](#options_color)
-    7. [styles](#options_styles)
-    8. [context](#options_context)
-    9. [errors.callback](#options_errors_callback)
-    10. [headersRegex](#options_headers_regex)
-    11. [health](#options_health)
-    12. [outbound.enabled](#options_outbound_enabled)
-    13. [traceHeaderName](#options_trace_header_name)
+    6. [bodyValuesMaxLength] (#options_body_values_max_length)
```

Comment:
🔧 I see that we already support `outbound.maxQueryChars`, `outbound.maxPathChars`, `outbound.maxHrefChars`
So for uniformity I would expect this to be `inbound.maxBodyValuesChars` and `outbound.maxBodyValuesChars`.
Can using only this config ensure a maximum body chars?

Reply:
Do you suggest splitting this into to options?

It doesn't ensure that the maximum body chars would be under 256K, but I don't expect payloads with many of those fields. I am thinking of putting the limit to something like 10K, so you need to have ~20 files to the payload

Reply:
> Do you suggest splitting this into to options?

Renaming as a first step to match the other configs. If it is not split similarly to other configs won't it be a bit weird as configuration? 
> 
> It doesn't ensure that the maximum body chars would be under 256K, but I don't expect payloads with many of those fields. I am thinking of putting the limit to something like 10K, so you need to have ~20 files to the payload

Ok makes sense